src/js/page_sections/Navmenu.js

```diff
@@ -106,20 +106,20 @@ class Navmenu extends Component {
 
             return (
               <li key={parentLink.id} onClick={this.props.toggleMenu}>
-                <Link to={getPathWithLangCode(`/topic/${parentLink.id}`)}
+                <Link to={`/topics/${parentLink.id}`}
                   className={this.getParentMenuItemClassName(parentLink)}
                 >
                   { parentLink.text }
                 </Link>
 
-                { serviceLinks.length && (
+                { !!serviceLinks && (
```

Comment:
if serviceLinks is a zero length array, this will evaluate to true. I think we need to check length specifically.

Reply:
I was seeing a zero length array return `0` onto the DOM. I saw some twitter thread talking about this quirk in rendering elements with a map function but now I can't find it. That thread suggested using double bang so that's where I got the inspiration. I promise this fixed the bug I was seeing. But I can roll it back since I can't defend my actions with any evidence. Either way, if the navmenu doesn't return any list items, we have bigger problems.